index.js

```diff
@@ -33,7 +33,9 @@ module.exports = (target, opts) => {
 	} else if (process.platform === 'win32' || isWsl) {
 		cmd = 'cmd' + (isWsl ? '.exe' : '');
 		args.push('/c', 'start', '""', '/b');
-		target = target.replace(/&/g, '^&');
+		if (target.indexOf(' ') === -1) {
```

Comment:
```suggestion
		if (!target.includes(' ')) {
```

Reply:
Can you also link to https://github.com/sindresorhus/open/pull/74#issuecomment-403681779 in a code comment to clarify what it does and why.